<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BibTexExportDialog</class>
 <widget class="QDialog" name="BibTexExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>File type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="selectedFileType">
          <item>
           <property name="text">
            <string>BibTeX</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>blg</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HTML</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PDF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ris</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rtf</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>xml</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>xslt</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select the file:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KUrlRequester" name="folder">
        <property name="mode">
         <set>KFile::File|KFile::LocalOnly</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Collection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Export Data from:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KComboBox" name="dataSelection"/>
      </item>
      <item>
       <widget class="QCheckBox" name="onlyReferences">
        <property name="text">
         <string>only References</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BibTexExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BibTexExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
